### ISSUE TITLE
Feature request: symbol meanings on hover

### ISSUE THREAD

**Issue by aonghas43** [mooseling/BananaDrum#8]
For some instruments where there is a choice of > 2 sounds (rep, timba), it might help to have the meaning of the symbol appear on hovering, or something similar. (A full legend to explain all symbols would be irritating to more experienced users, and besides there is probably not the space for it)

**Comment by mooseling** [comment]
Interesting! There's a lot to think about here.
First off, can I ask what made you think of this? I am trying to understand who this an issue for, and what problems it causes. I can see that maybe a new player will be intrigued by the various things you can do with, eg, a repinique, but be frustrated not to be given any words. It makes it hard to look things up or talk about them.
I'm always extremely hesitant to add anything to the UI. I want the user interface to be simple, playful, and inviting. A tooltip is nice because it doesn't add clutter, but I generally find tooltips annoying. It's visual noise, distraction. I also think that the text-free presentation of Banana Drum is part of its charm and purpose.
I do think there may be a place for explaining the different sounds, but I don't think I know the answer yet.
Ableton Live (to me, a rare example of amazing software) has a nice solution for this. They have a panel in a bottom corner that you can open up if you want, and it will show an explanation of whatever you mouse over. This is overkill for Banana Drum, Ableton Live does millions of things.
I'm thinking maybe we solve this incidentally in a future change. For example, I will probably overhaul the "add instrument" UI at some point, and more instrument info could go there. Or, I'm thinking of a quick-compose feature where you can type out a rhythm. This will involve some graphical stuff that could contain the names of the sounds.

**Comment by aonghas43** [comment]
I am new to samba, and the various different sounds for the rep and timba
are a bit of  mystery.  I understand the appeal of a symbol.

My particular use case is "new player trying to transcribe a whole section
from a chart to practice", which is perhaps a bit niche, but I am not sure
if there are well-understood conventions for symbols for the different
sounds on timba and hep.

Big X and little x, and open and closed circle for other instruments are
easy to get, likewise /// for a buzz.

I figured the symbols out which I didn't understand by looking in the
source on github, so I suppose I have solved my own problem, but I still
think there might be room for some  symbols to be misunderstood. Some
people for example may be more familiar with letters. Maybe worth asking
the band the question about how things are represented on a chart ?

**Comment by mooseling** [comment]
Aha, I see! That's a very valid use case. Interesting. I think you're right that this should be solved.

At Arco Iris, the charts tend to use letters representing English words or sounds. I think I've seen "O" for open-tone (on timbau, but also rep with the left hand), "S" for slap (ditto), "Zz" for buzz. I decided not to take this direction with banana drum because not everyone speaks English! As far as I know, banana drum is being used in at least the UK, Germany, and Brazil :D

I've added the issue to Trello and will give it a think. I will close this feature request for now. Thanks!
